src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,16 @@
+.catalog__page {
```

Comment:
even though you are not styling the catalog class, you can still use the goodies of scss indentation and the & sign.